<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QDialog" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="editorSettings">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0" colspan="3">
        <widget class="QSpinBox" name="editorTabWidth">
         <property name="suffix">
          <string> spaces</string>
         </property>
         <property name="prefix">
          <string>Tab width is </string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>80</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="editorFontName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="editorReplaceTabs">
         <property name="text">
          <string>Replace tabs with spaces</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Editor Font</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="editorStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="editorFontSize">
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>72</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Color style</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="saveOnActionTarget">
         <property name="text">
          <string>Save editor contents on target action</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="CPPTextEditor" name="codeEditor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolsSettings">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Workspace PATH</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="workspacePath"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="projectPathSetButton">
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/images/actions/document-open.svg</normaloff>:/images/actions/document-open.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Additional PATHs</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="3">
           <widget class="QListView" name="additionalPathList"/>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="tbPathAdd">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/images/actions/list-add.svg</normaloff>:/images/actions/list-add.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="tbPathRm">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/images/actions/list-remove.svg</normaloff>:/images/actions/list-remove.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>177</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TemplateManager" name="templateSettings">
      <attribute name="title">
       <string>Templates</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="proxySettings">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>HTTP proxy for network access</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="noProxy">
              <property name="text">
               <string>None</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="systemProxy">
              <property name="text">
               <string>System proxy</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="userProxy">
              <property name="text">
               <string>Custom proxy</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelHost">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxyHost">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>localhost</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelPort">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxyPort">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>3128</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="useAutentication">
            <property name="title">
             <string>Proxy authentication</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelUsername">
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="username">
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelPassword">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="password">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="behaviorSettings">
      <attribute name="title">
       <string>Behavior</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="autoUpdateProjectTmplates">
         <property name="text">
          <string>Check for project templates updates at start up</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Log View Font</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="loggerFontName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="fontFilters">
          <set>QFontComboBox::MonospacedFonts</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="loggerFontSize">
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>72</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid;
border-bottom-color: rgb(0, 0, 0, 96);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Format Style</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="formatterStyle"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Format extra paramters:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="formatterExtra"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="useDevelopment">
         <property name="text">
          <string>Use in-progress/development characteristics</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPPTextEditor</class>
   <extends>QWidget</extends>
   <header>cpptexteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemplateManager</class>
   <extends>QWidget</extends>
   <header>templatemanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>useAutentication</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
